### PR TITLE
fix docker build

### PR DESCRIPTION
**Pull request by beauraF**
Because of this line, the docker build production of `doctolib/doctolib` has failed ; leading to the revert of your PR. I want to do my best, to have this running the whole week-end on staging, but I don't have write access on this repository . . . So I have a fork, and will merge a PR that use my fork for now. I'll let you fix that on Monday.

### CODE REVIEW

**Comment by beauraF** [comment]
https://github.com/doctolib/doctolib/pull/145617

**Comment by beauraF** [comment]
I though I would by able to merge it, with some piraterie ... but it seems the system is smarter than me: https://github.com/doctolib/doctolib/actions/runs/7150573120. Sorry I won't be able to get it on staging, I haven't admin access :(

**Comment by beauraF** [comment]
Ok I have been able to merge it by removing a small part that required a codeowners approval: https://github.com/doctolib/doctolib/pull/145617. I let you take over on Monday :D

**Review by pimpin — APPROVED** [review]

**Pull request merged**
